map/map_loader/CMakeLists.txt

```
@@ -39,12 +39,21 @@ rclcpp_components_register_node(lanelet2_map_visualization_node
   EXECUTABLE lanelet2_map_visualization
 )
 
+set(FOLDERS_TO_SHARE launch config)
+
+if(BUILD_TESTING)
+  add_ros_test(
+    test/lanelet2_map_loader_launch.test.py
+    TIMEOUT "30"
+  )
+  list(APPEND FOLDERS_TO_SHARE test)
```

Is it possible to use `test/data` here? :thinking: 

```suggestion
  list(APPEND FOLDERS_TO_SHARE test/data)
```

---

Done: [3d6329f](https://github.com/autowarefoundation/autoware.universe/pull/1056/commits/3d6329fa424d75ba5551988d4bd40b5484767988)